components/faunadb/faunadb.app.js

```diff
@@ -0,0 +1,61 @@
+// FaunaDB app file
+const faunadb = require("faunadb");
+const q = faunadb.query;
+
+module.exports = {
+  type: "app",
+  app: "faunadb",
+  propDefinitions: {
+    collection: {
+      type: "string",
+      label: "Collection",
+      description: "The collection you'd like to watch for changes",
+      optional: false,
+      async options() {
+        return await this.getCollections();
+      },
+    },
+  },
+  methods: {
+    // Fetches events (changes) for all documents in a collection
+    // made after a specific epoch timestamp
+    getClient() {
+      return new faunadb.Client({ secret: this.$auth.secret });
+    },
+    async getCollections() {
```

Comment:
Does this get used by anything? Should it be called `getCollectionIds`? (and the var inside should maybe be renamed too?)

Reply:
Oh nvm ^F didn't work... can you return [{label,value}] ?

Reply:
(i see it in async options now)

Reply:
done